doc-content/enterprise-only/fuse/fuse-con.adoc

```diff
@@ -20,11 +20,21 @@ You can install {FUSE_LONG} on a separate instance of {EAP_LONG} and integrate i
 
 [IMPORTANT]
 ====
-For the version of {FUSE_LONG} that {PRODUCT} {ENTERPRISE_VERSION} supports, see 
+For the version of {FUSE_LONG} that {PRODUCT} {ENTERPRISE_VERSION} supports, see
 ifdef::PAM[]
 https://access.redhat.com/articles/3405381[{PRODUCT} 7 Supported Configurations].
 endif::[]
 ifdef::DM[]
 https://access.redhat.com/articles/3354301[{PRODUCT} 7 Supported Configurations].
 endif::[]
 ====
+
+[NOTE]
+====
+You can also install {FUSE_LONG} on SpringBoot. {PRODUCT} provides no special integration for this scenario. However, your application that runs on {FUSE_LONG} on SpringBoot can use the `kie-server-client` library to communicate with {PRODUCT} services that run on a {KIE_SERVER}. The following limitations apply, because of library version conflicts between {FUSE_LONG} and {PRODUCT}:
```

Comment:
[style] Correct "SpringBoot" to "Spring Boot", remove "also", and maybe rephrase to simplify. For example:

You can install {FUSE_LONG} on Spring Boot. You can use the `kie-server-client` library to enable communication between applications running on {FUSE_LONG} on {PRODUCT} with {PRODUCT} services running on {KIE_SERVER}. There are `kie-server-client` library version conflicts between {FUSE_LONG} and {PRODUCT} that cause the following limitations:

Reply:
I do believe that the "no special integration" part is important.